l1-contracts/bootstrap.sh

```diff
@@ -131,6 +131,84 @@ function gas_report {
   mv gas_report.new.json gas_report.json
 }
 
+function bench {
+  local hash=$(git rev-list -n 1 ${AZTEC_CACHE_COMMIT:-HEAD})
+
+  rm -rf bench-out && mkdir -p bench-out
+  if cache_download l1-gas-bench-results-$hash.tar.gz; then
+    return
+  fi
+
+  # Run the gas benchmark to generate the markdown file
+  gas_benchmark
+
+  # Extract gas values from gas_benchmark.md and create JSON output
+  awk '
+  function trim(s) {
+    sub(/^[ \t]+/, "", s);
+    sub(/[ \t]+$/, "", s);
+    return s;
+  }
+  BEGIN {
+    print "[";
+    first = 1;
+  }
+  /^[a-zA-Z]/ {
+    if ($1 != "Function" && $1 != "-------------------------") {
+      # Split the line into columns and clean them
+      n = split($0, cols, "|");
+      for (i = 1; i <= n; i++) {
+        cols[i] = trim(cols[i]);
+      }
+
+      # Only process Max rows
+      if (cols[2] == "Max") {
+        # Get the function name
+        func_name = cols[1];
+
```

Comment:
Why awk?

Reply:
It did the job, was available and mr robot was doing well with it